### PR TITLE
Changes scroll fail messages, adds some more

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- I told you the PR would come _evil laugh_
-
-

Review please

### CODE REVIEW

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 1]
```diff
@@ -343,28 +343,32 @@
...
+  "minecolonies.scroll.needcolony": "Please register this to a colony first by shift + right-clicking a hut block.",
+  "minecolonies.scroll.noguardbuilding": "Please register this to a Guard or Barracks Tower by shift + right-clicking it.",
+  "minecolonies.scroll.wrongdim": "This scroll can only be used in the same dimension as the colony.",
+  "minecolonies.scroll.registered": "Registered scroll with colony %s.",
   "minecolonies.scroll.failed1": "The unstable magic of the scroll backfired!",
-  "minecolonies.scroll.failed2": "Whoops, what happened now?!",
-  "minecolonies.scroll.failed3": "This scroll needs to go back to the drawing board!",
-  "minecolonies.scroll.failed4": "The scroll doesnt seem to be made by the best magician out there!",
+  "minecolonies.scroll.failed2": "Whoops!",
+  "minecolonies.scroll.failed3": "Uh oh!",
```
Comment (ShadowProtocol) [inline src/main/resources/assets/minecolonies/lang/en_us.json]: 3 was kinda fun, personally I'd leave it as is

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 2]
```diff
@@ -343,28 +343,32 @@
...
+  "minecolonies.scroll.failed4": "This scroll doesn't seem to be made by a good magician!",
```
Comment (ShadowProtocol) [inline src/main/resources/assets/minecolonies/lang/en_us.json]:
````suggestion
  "minecolonies.scroll.failed4": "It seems I was made by an amateur magician!",
````

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 3]
```diff
@@ -343,28 +343,32 @@
...
   "item.minecolonies.scroll_area_tp": "Ultrasafe colony group-teleport scroll",
-  "item.minecolonies.scroll_area_tp.tip": "Teleport yourself and nearby players to the colony",
+  "item.minecolonies.scroll_area_tp.tip": "Teleports yourself and nearby players to the colony",
   "item.minecolonies.scroll_buff": "Sacred Scroll of Regeneration",
   "item.minecolonies.scroll_buff.tip": "Applies regeneration to citizens around you, at least most of the time...",
   "item.minecolonies.scroll_guard_help": "Spatial guard reinforcement scroll",
   "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time, wololo!",
   "item.minecolonies.scroll_highlight": "Worker-where-are-you scroll",
-  "item.minecolonies.scroll_highlight.tip": "Sneak-Use on a minecolonies building, to highlight and speed up its workers",
+  "item.minecolonies.scroll_highlight.tip": "Sneak + right-click on a hut block to highlight and speed up its worker(s)",
```
Comment (ShadowProtocol) [inline src/main/resources/assets/minecolonies/lang/en_us.json]: I really don't mind either way, but I thought I'd just comment that sneak-use is more accurate from a usability perspective as some ~~strange~~ players might change the controls.

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 4]
```diff
@@ -343,28 +343,32 @@
   "item.minecolonies.scroll.colony.tip": "Colony: %s",
   "item.minecolonies.scroll.colony.none": "None, set one with sneak + right-click",
   "item.minecolonies.scroll_area_tp": "Ultrasafe colony group-teleport scroll",
-  "item.minecolonies.scroll_area_tp.tip": "Teleport yourself and nearby players to the colony",
+  "item.minecolonies.scroll_area_tp.tip": "Teleports yourself and nearby players to the colony",
```
Comment (ShadowProtocol) [inline src/main/resources/assets/minecolonies/lang/en_us.json]: It was describing what the scroll does, not what you do. It is implying the subject:
"[the scroll] teleports yourself ..."
But you're right that it sounds a little odd
````suggestion
  "item.minecolonies.scroll_area_tp.tip": "Teleports you and nearby players to the colony",
````

**Review by ShadowProtocol — COMMENTED** [review]
I didn't get time to nitpick someaddons, so you're just getting enough for both of you :smile: 

Another thing is I think the names of the scrolls could use some work, too. Some ideas:
Colony Portal Scroll
Colony Team Portal Scroll
Reinforcements Portal Scroll
Worker Seeker Scroll

File: src/main/resources/assets/minecolonies/lang/en_us.json (hunk 3 above)
Comment (ravenbuilder934) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: Looking at you, Kartijan!

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 5]
```diff
@@ -338,33 +338,37 @@
...
+  "item.minecolonies.scroll_guard_help": "Reinforcements Scroll",
+  "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time. Wololo!",
+  "item.minecolonies.scroll_highlight": "Worker Locator Scroll",
+  "item.minecolonies.scroll_highlight.tip": "Sneak + use on a hut block to highlight and speed up its worker(s)",
   "minecolonies.scroll.nopermission": "You do not have the required permissions for this colony.",
-  "minecolonies.scroll.needcolony": "Please register this to a colony first with shift + right-clicking a building.",
-  "minecolonies.scroll.noguardbuilding": "Please register this to a guard building by shift + right-clicking it.",
-  "minecolonies.scroll.wrongdim": "Can only be used in the same dimension as the colony.",
-  "minecolonies.scroll.registered": "Registered scroll with colony %s",
+  "minecolonies.scroll.needcolony": "Please register this to a colony first by sneak + using a hut block.",
```
Comment (ShadowProtocol) [inline src/main/resources/assets/minecolonies/lang/en_us.json]: We're using the scroll, not the hut block :)
```suggestion
  "minecolonies.scroll.needcolony": "Please register this to a colony first by sneak + using it on a hut block.",
```

**Review by ShadowProtocol — COMMENTED** [review]
looking good.

**Review by ShadowProtocol — DISMISSED** [review]

File: src/main/resources/assets/minecolonies/lang/en_us.json [hunk 6]
```diff
@@ -338,33 +338,37 @@
...
+  "item.minecolonies.scroll_guard_help": "Reinforcements Scroll",
+  "item.minecolonies.scroll_guard_help.tip": "Call your guards to help you out for a limited time. Wololo!",
+  "item.minecolonies.scroll_highlight": "Worker Locator Scroll",
+  "item.minecolonies.scroll_highlight.tip": "Sneak + use on a hut block to highlight and speed up its worker(s)",
   "minecolonies.scroll.nopermission": "You do not have the required permissions for this colony.",
-  "minecolonies.scroll.needcolony": "Please register this to a colony first with shift + right-clicking a building.",
-  "minecolonies.scroll.noguardbuilding": "Please register this to a guard building by shift + right-clicking it.",
-  "minecolonies.scroll.wrongdim": "Can only be used in the same dimension as the colony.",
-  "minecolonies.scroll.registered": "Registered scroll with colony %s",
+  "minecolonies.scroll.needcolony": "Please register this to a colony first by sneak + using it on a hut block.",
```
Comment (Raycoms) [inline src/main/resources/assets/minecolonies/lang/en_us.json]: should probably be "by sneaking" + "using"

Comment (ShadowProtocol) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: You're on to something there. I was just reading like it like there was a hyphen, so you could do it ``sneak-using`` or ``sneaking + using`` I guess.

Comment (Nightenom) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: to remove confusion I would replace this with the item name, might be also worth to say which hut block
_requires code changes_

**Comment by someaddons** [comment]
I like the old names still better than those suggested here, and I think it should be sneak + rightclick not using. The scroll does tell you which building, when it requires a specific one.

**Comment by someaddons** [comment]
Also I did not want to add those you're trying to add now, I've left them out on purpose!

File: src/main/resources/assets/minecolonies/lang/en_us.json (hunk 6 above)
Comment (ShadowProtocol) [reply src/main/resources/assets/minecolonies/lang/en_us.json]: Normally I would agree, but the names of the items are somewhat lengthy, I think maybe ``sneak-using the scroll on a hut block.`` would suffice.

**Comment by ShadowProtocol** [comment]
> I like the old names still better than those suggested here, and I think it should be sneak + rightclick not using. The scroll does tell you which building, when it requires a specific one.

You're combining an in-game action with an out-of-game control. Not really a proper thing to do. But it does make sense and is potentially the clearest so as I stated earlier I don't really mind.

As for the names:
Ultrasafe - the players have no other item to compare this to, unless you plan on making safe and unsafe scrolls too. So it becomes a meaningless adjective. Especially as you already say it's safe in the tooltip.
Spatial is an anachronism, I'm pretty sure they did not think of teleporting in this sci-fi manner in the Colonial Era.
Worker-where-are-you - I can very much hear your voice in my head when I see this, but it doesn't sound all that official or mystical. Kinda fun though so I get why you like that one.
They should at least be Title Case.

And yes I am being generally pedantic as usual; sorry about that.

**Comment by Raycoms** [comment]
I think the "ultrasave" is meant to be sarcastic.

**Comment by ShadowProtocol** [comment]
> I think the "ultrasave" is meant to be sarcastic.

I thought that might be the case, in which it's completely lost on most people, especially those who have more trouble with English.

**Comment by someaddons** [comment]
Ultrasafe is an exaggeration aswell as a game reference to another popular game, having a similar item also having ultrasafe in its name with similar function and drawbacks

**Comment by ShadowProtocol** [comment]
Oh I think it's pretty humorous, but that still doesn't disqualify any previous points.
I'm just trying to think of all the people who barely know how to build their first hut let alone warp ~~safely~~ using a mystical scroll. But their loss I guess, I've done my job. Can't sue me now.

**Comment by someaddons** [comment]
Those people wont get that item in the first place, it takes an enchanter and understanding a postbox to obtain it

**Comment by ShadowProtocol** [comment]
You're not wrong, but that was not my point. I meant it might take some people a while to make the connection that the item name is facetious. But anyways let's just go with it I guess.

**Comment by ravenbuilder934** [comment]
> Also I did not want to add those you're trying to add now, I've left them out on purpose!

Why? What purpose?

> You're not wrong, but that was not my point. I meant it might take some people a while to make the connection that the item name is facetious. But anyways let's just go with it I guess.

How about I put it in quotes? so "Ultrasafe" colony teleport scroll? Then it's easier to tell it's sarcasm

**Comment by someaddons** [comment]
because I don't think they fit the overall theme of the items well enough.
No need for quotes, its not necessary for all players to understand it right away

**Comment by ravenbuilder934** [comment]
> because I don't think they fit the overall theme of the items well enough.

Why does 404 fit but not 418?

**Comment by someaddons** [comment]
because 404 is better known and I also dont want to overload it too muc with similar messages thus I feel it does not fit in

**Comment by ravenbuilder934** [comment]
> 404 is better known

That's the fun of it! Players will google 418 I'm a teapot, and be introduced to this wonderful nerdy joke 😄 

> I also dont want to overload it too muc with similar messages

If a fail is rare, and each fail message is chosen randomly, the similar messages shouldn't be an issue.

**Pull request closed without merging**